src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -732,7 +732,8 @@ module.exports.external = [
         name: 'generateUUID',
         description: 'Generates a guid based on a URL: generateUUID url',
         func: function (urlNamespace) {
-            return uuidv3(''.concat(urlNamespace), uuidv3.URL);
+            const content = ''.concat(urlNamespace).replace(/(\r|\n|\r\n|\\r|\\n|\\r\\n)/gm, '');
```

Comment:
Users may get different ID result after using this new code. We should create a bug and let Ranvijay know this change?

Reply:
Right, and an email has been sent to Ranvijay.